### PR TITLE
[T-215] Reviews - BE part

### PR DESCRIPTION
**Pull request by lakatop**

### CODE REVIEW

File: src/backend/core/review/views.py [hunk 1]
```diff
@@ -1,3 +1,77 @@
...
+            return Response(status=403)
+
+        if Review.objects.filter(product=product, order=order).exists():
+            return Response(status=403)
+
+        review = Review.objects.create(product=product, order=order)
+        return Response({"token": review.token}, status=201)
+
+
+class ReviewDetailView(APIView):
```
[the remark this answers is not in the document]
Comment (lakatop) [reply src/backend/core/review/views.py:37]: Yeah thats related to a question I was about to ask - do we want to allow admin/business user to modify the review? Might be a way to mislead a customer :D

[the remark this answers is not in the document]
Comment (VL-CZ) [reply src/backend/core/review/views.py:37]: Deletes definitely make sense (if the comment is e.g. inappropriate), but updates are useless IMO

[the remark this answers is not in the document]
Comment (VL-CZ) [reply src/backend/core/review/views.py:37]: btw. I'd consider renaming the view to smth like `ReviewDetailDashboardView` to include the word `dashboard` in it.
It's definitely not a requirement, rather a convention that we use for distinguishing between dashboard/storefront views 🙂

[the remark this answers is not in the document]
Comment (michalpulpan) [reply src/backend/core/review/views.py:37]: Btw. be careful with `RetrieveUpdateDestroyApiView`, it doesn't define `DELETE` HTTP method by default, but it's called `DESTROY`. I was caught in this trap for some "non-trivial amount of time" 😂.

**Review by michalpulpan — APPROVED** [review]
Lookin good 🔥

**Review by VL-CZ — APPROVED** [review]

**Pull request closed without merging**